docs/docs/start/getting-started.mdx

```diff
@@ -1,19 +1,249 @@
 ---
+title: Getting Started
+sidebar_position: 1
 sidebar_custom_props:
-  icon: TbBolt
+  icon: TbDeviceDesktop
```

Comment:
Isn't TbDeviceDesktop already used elsewhere? I didn't check but assume it would be for local setup. Make sure it isn't used twice, that would be ugly :)